### ISSUE TITLE
Training set should be test set in boostrapping lesson

### ISSUE THREAD

**Issue by ttohya** [carpentries-incubator/machine-learning-novice-python#26]
In the following section we evaluate the performance on the training set.

https://github.com/carpentries-incubator/machine-learning-novice-python/blob/8b353791dfd982f8c2eb79174ce6e575fb6d2512/_episodes/07-bootstrapping.md?plain=1#L71-L77

 I think we should use the test set here.
